lib/ococo/meta/process_params.js

```diff
@@ -0,0 +1,40 @@
+'use strict'
+
+module.exports = (data) => {
+  const params = { ...data }
+
+  if (params._symbol) {
+    params.symbol = params._symbol
+    delete params._symbol
+  }
+
+  if (!params._futures) {
+    delete params.lev
+  }
+
+  if (!params.cancelDelay) {
+    params.cancelDelay = 1500
+  }
+
+  if (!params.submitDelay) {
+    params.submitDelay = 5000
```

Comment:
Why so high? Ping pongs default is only 2000:

```
if (!_isFinite(params.cancelDelay)) {
    params.cancelDelay = 1000
  }

  if (!_isFinite(params.submitDelay)) {
    params.submitDelay = 2000
  }
```

Reply:
Fixed; copy-paste mistake, it was also accessing the wrong key for the parameter. Nice catch :bowing_man: